src/autoCollection/metrics/httpMetricsInstrumentation.ts

```diff
@@ -51,20 +65,28 @@ export class HttpMetricsInstrumentation extends InstrumentationBase {
     return [this._getHttpDefinition(), this._getHttpsDefinition()];
   }
 
-  private _htppRequestDone(metric: IHttpMetric) {
+  private _httpRequestDone(metric: IHttpStandardMetric) {
     // Done could be called multiple times, only process metric once
     if (!metric.isProcessed) {
+      metric.isProcessed = true;
```

Comment:
Curious as to why `_httpRequestDone` is called multiple times for the same request?

Reply:
It could be called multiple times for specific cases, like request timeout, abort, etc.